docs/user/streaming-data.md

```diff
@@ -73,4 +73,19 @@ obj = s3.get_object(Body=csv_buffer.getvalue(), Bucket="my-bucket", Key="my/doc.
 reader = PdfReader(BytesIO(obj["Body"].read()))
 ```
 
-It works similarly for Google Cloud Storage ([example](https://stackoverflow.com/a/68403628/562769)).
+To use with Google cloud storage
+
+```python
+from io import BytesIO
+from google.cloud import storage
+import os
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "Set this up either as secret or json file downloaded from gcloud"
```

Comment:
Please add a comment instead which indicates that the user has to set this environment variable.

Reply:
Ok will do thank you